extensions-core/datasketches/src/test/java/org/apache/druid/query/aggregation/datasketches/theta/sql/ThetaSketchSqlAggregatorTest.java

```diff
@@ -1018,8 +1034,160 @@ public void testGroupByAggregatorDefaultValuesFinalizeOuterSketches()
   @Test
   public void testThetaSketchIntersectOnScalarExpression()
   {
-    assertQueryIsUnplannable("SELECT THETA_SKETCH_INTERSECT(NULL, NULL) FROM foo",
+    assertQueryIsUnplannable(
+        "SELECT THETA_SKETCH_INTERSECT(NULL, NULL) FROM foo",
         "Possible error: THETA_SKETCH_INTERSECT can only be used on aggregates. " +
-            "It cannot be used directly on a column or on a scalar expression.");
+        "It cannot be used directly on a column or on a scalar expression."
+    );
+  }
+
+  @Test
+  public void testThetaSketchEstimateAsVirtualColumn()
+  {
+    testQuery(
+        "SELECT"
+        + " THETA_SKETCH_ESTIMATE(thetasketch_dim1)"
+        + " FROM foo",
+        ImmutableList.of(
+            newScanQueryBuilder()
+                .dataSource(CalciteTests.DATASOURCE1)
+                .intervals(querySegmentSpec(Filtration.eternity()))
+                .virtualColumns(new ExpressionVirtualColumn(
+                    "v0",
+                    "theta_sketch_estimate(\"thetasketch_dim1\")",
+                    ColumnType.DOUBLE,
+                    MACRO_TABLE
+                ))
+                .resultFormat(ScanQuery.ResultFormat.RESULT_FORMAT_COMPACTED_LIST)
+                .columns("v0")
+                .context(QUERY_CONTEXT_DEFAULT)
+                .build()
+        ),
+        ImmutableList.of(
+            NullHandling.replaceWithDefault() ? new Object[]{null} : new Object[]{0.0D},
```

Comment:
What causes the difference here?  It would appear that this and HLL are not consistent with each other and I'm curious why.